### PR TITLE
Chronos: userguide revision(Forecaster, TSDataset, Useful functionalities)

### PR DESCRIPTION
**Pull request by TheaperDeng**
preview: https://junweitestdoc.readthedocs.io/en/autots-userguide-new/doc/Chronos/Overview/chronos.html
This PR mainly revised Section 4, 5, 7

### CODE REVIEW

**Review by yushan111 — APPROVED** [review]
LGTM

**Comment by yushan111** [comment]
BTW, shall we move the data processing (`TSDataset`) part in front of "Forecasting"? Since data preparation is always the first step and user could have a better understanding on 4.2.1

File: docs/readthedocs/source/doc/Chronos/Overview/chronos.md [hunk 1]
```diff
@@ -57,163 +82,118 @@ Recent years, common deep learning architectures (e.g. RNN, CNN, Transformer, et
...
-2019-06-06  1.2     1                2
-2019-06-07  2.30    2                1
+#### **4.2 Use AutoTS Pipeline**
+For AutoTS Pipeline, we will leverage `AutoTSEstimator`, `TSPipeline` and preferably `TSDataset`. A typical usage of AutoTS pipeline basically contains 3 steps.
+1. Prepare a `TSDataset` or customized data creator.
+2. Init a `AutoTSEstimator` and call `.fit()` on the data.
+3. Use the returned `TSPipeline` for further development.
+```eval_rst
+.. warning::
+    `AutoTSTrainer` workflow has been deprecated, no feature updates or performance improvement will be carried out. Users of `AutoTSTrainer` may refer to Chronos API doc.
```
[the remark this answers is not in the document]
Comment (TheaperDeng) [reply docs/readthedocs/source/doc/Chronos/Overview/chronos.md]: fixed

**Comment by TheaperDeng** [comment]
> BTW, shall we move the data processing (`TSDataset`) part to the front of "Forecasting"? Since data preparation is always the first step and user could have a better understanding on 4.2.1

Good idea, changed

**Comment by TheaperDeng** [comment]
http://10.112.231.51:18888/job/ZOO-PR-Validation/6438/
http://10.112.231.51:18888/job/ZOO-PR-Python-ExampleTests-py37-Spark3/1988/
http://10.112.231.51:18888/job/ZOO-PR-Scala-Spark-2.1/6531/

**Pull request merged**